docs/installation.md

```
@@ -169,14 +162,26 @@ Agent Zero provides a comprehensive settings interface to customize various aspe
 
 ![chat model settings](res/setup/settings/2-chat-model.png)
 
+> [!IMPORTANT]
+> **Model naming is provider-specific.** Use `gpt-4.1` for OpenAI, but use `openai/gpt-4.1` for OpenRouter. If you see “Invalid model ID,” verify the provider and naming format.
+
```

markdownlint-cli2 (0.18.1)

Blank line inside blockquote

(MD028, no-blanks-blockquote)